src/parser/vast_parser.js

```diff
@@ -209,6 +211,20 @@ export class VASTParser extends EventEmitter {
       }
     }
 
+    const vastAdsLength = vastResponse.ads.length;
+    const lastAddedAds = vastResponse.ads[vastAdsLength - 1];
+    // if in child nodes we have only one ads
+    // and wrapperSequence is defined
+    // and this ads doesn't already have sequence
+    if (
+      countParsedAds === 1 &&
+      options.wrapperSequence !== undefined &&
+      options.wrapperSequence !== null &&
+      !lastAddedAds.sequence
```

Comment:
we need to check for `lastAddedAds` existance before, since it can be `undefined` we may have `Uncaught TypeError: Cannot read property 'sequence' of undefined`

So just need to add 
```Javascript
lastAddedAds &&
!lastAddedAds.sequence
```